src/test/java/com/pusher/client/channel/impl/PrivateEncryptedChannelImplTest.java

```diff
@@ -0,0 +1,122 @@
+package com.pusher.client.channel.impl;
+
+import com.pusher.client.AuthorizationFailureException;
+import com.pusher.client.Authorizer;
+import com.pusher.client.connection.impl.InternalConnection;
+import com.pusher.client.util.Factory;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Matchers;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PrivateEncryptedChannelImplTest {
+
+    private String authorizer_valid = "{\"auth\":\"636a81ba7e7b15725c00:3ee04892514e8a669dc5d30267221f16727596688894712cad305986e6fc0f3c\",\"shared_secret\":\"iBvNoPVYwByqSfg6anjPpEQ2j051b3rt1Vmnb+z5doo=\"}";
+    private String authorizer_missingAuthKey = "{\"shared_secret\":\"iBvNoPVYwByqSfg6anjPpEQ2j051b3rt1Vmnb+z5doo=\"}";
+    private String authorizer_missingSharedSecret = "{\"auth\":\"636a81ba7e7b15725c00:3ee04892514e8a669dc5d30267221f16727596688894712cad305986e6fc0f3c\"}";
+    private String authorizer_malformedJson = "potatoes";
+
+    private InternalConnection internalConnection;
+    private Authorizer authorizer;
+    private Factory factory;
+
+    @Before
+    public void setup() {
```

Comment:
```suggestion
    public void setUp() {
```
as it's two words (as a verb phrase), also [consistency with the existing code](https://github.com/pusher/pusher-websocket-java/blob/9745b3dc358b2fe3f111ea17b882a8a879c8c049/src/test/java/com/pusher/client/channel/impl/PrivateChannelImplTest.java#L34)

Reply:
I found with the suggestion to annotate the mocked things, I didn't need a setUp method anymore :)